libcontainer/cgroups/fs2/devices.go

```diff
@@ -26,11 +28,21 @@ func isRWM(perms devices.Permissions) bool {
 	return r && w && m
 }
 
-// the logic is from crun
-// https://github.com/containers/crun/blob/0.10.2/src/libcrun/cgroup.c#L1644-L1652
 func canSkipEBPFError(cgroup *configs.Cgroup) bool {
+	// If we're running in a user namespace we can ignore eBPF rules because
+	// we're already restricted to not be able to mknod(2) by default.
```

Comment:
s/mknod/bpf/ ?

Reply:
I did mean `mknod(2)` because that's one of the things the devices rules block, but now that you mention it `bpf(2)` being blocked is the more direct reason why we ignore this (and the devices rules block other things that are non-trivial to block outside of cgroups).